app/models/collection.rb

```diff
@@ -121,7 +122,10 @@ def to_open_search_collections(document, hits, params, url, for_atom = true, cwi
           add_link_as_child(doc, node, href, 'text/html', 'enclosure', short_name)
         end
 
-        if provider_osdd_link.present?
```

Comment:
I believe the form of a provider OSDD link has changed (https://bugs.earthdata.nasa.gov/browse/ECSE-995) does that require a change in the present? method or does https://bugs.earthdata.nasa.gov/browse/CMR-7178 abstract that away from you?

Reply:
The provider_osdd_link is still using the tags to identify it, but this `collection_specific_osdd_link` is using the new rel value of `'http://esipfed.org/ns/fedsearch/1.1/search#'` to identify it. And if I understood the change required correctly, we are looking first for the new link type first and using that if it exists, and if not they we will look for the older tag identified link to provide. Did I misunderstand anything?